it.mijorus.gearlever.json

```diff
@@ -0,0 +1,75 @@
+{
+    "app-id" : "it.mijorus.gearlever",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "44",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "gearlever",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--device=dri",
+
+        "--filesystem=xdg-data/applications:rw",
+        "--filesystem=xdg-data/icons:ro",
+        "--filesystem=home:rw",
+
+        "--talk-name=org.freedesktop.FileManager1",
+        "--talk-name=org.freedesktop.Flatpak",
+        "--talk-name=org.freedesktop.portal.OpenURI",
+        "--talk-name=org.freedesktop.portal.OpenFile",
```

Comment:
also I don't think you need to add the portal name to the list. (lines 19 and 20)